Dockerfile

```diff
@@ -1,6 +1,9 @@
 # syntax=docker/dockerfile:1
 FROM --platform=linux/amd64 synthetixio/docker-e2e:18.16-ubuntu as base
 
+# Prefer IPv4 for DNS resolution
+ENV NODE_OPTIONS=--dns-result-order=ipv4first
```

Comment:
I wonder whether there's a way to disable ipv6 for the whole Docker daemon. Node doesn't seem like the right level of abstraction for compatibility with the GH Action runner

Reply:
Shall we try [this solution](https://github.com/cypress-io/cypress/issues/27962#issuecomment-1938612795) if you recommend it?

We could pass the [host](https://vitejs.dev/config/server-options#server-host) flag to `vite`. This will allow the development server to accept connections from all network interfaces, and we won't need to disable IPv6 or set a priority for IPv4.